commit e9718b4bd3e4234ffd5a4907a903367fc483c843
Author: Michael Paquier <michael@paquier.xyz>
Date:   Mon Oct 16 09:36:31 2023 +0900

    Fix code indentation violations in e83d1b0c40cc
    
    koel has not reported this one yet, I have just bumped on it while
    looking at a different patch.

diff --git a/src/backend/commands/event_trigger.c b/src/backend/commands/event_trigger.c
index 0b08552fd7a..bf47b0f6e26 100644
--- a/src/backend/commands/event_trigger.c
+++ b/src/backend/commands/event_trigger.c
@@ -170,13 +170,13 @@ CreateEventTrigger(CreateEventTrigStmt *stmt)
 		validate_ddl_tags("tag", tags);
 	else if (strcmp(stmt->eventname, "table_rewrite") == 0
 			 && tags != NULL)
 		validate_table_rewrite_tags("tag", tags);
 	else if (strcmp(stmt->eventname, "login") == 0 && tags != NULL)
-			ereport(ERROR,
-					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-					 errmsg("Tag filtering is not supported for login event trigger")));
+		ereport(ERROR,
+				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+				 errmsg("Tag filtering is not supported for login event trigger")));
 
 	/*
 	 * Give user a nice error message if an event trigger of the same name
 	 * already exists.
 	 */
@@ -388,12 +388,12 @@ SetDatatabaseHasLoginEventTriggers(void)
 
 	/*
 	 * Use shared lock to prevent a conflit with EventTriggerOnLogin() trying
 	 * to reset pg_database.dathasloginevt flag.  Note, this lock doesn't
 	 * effectively blocks database or other objection.  It's just custom lock
-	 * tag used to prevent multiple backends changing pg_database.dathasloginevt
-	 * flag.
+	 * tag used to prevent multiple backends changing
+	 * pg_database.dathasloginevt flag.
 	 */
 	LockSharedObject(DatabaseRelationId, MyDatabaseId, 0, AccessExclusiveLock);
 
 	tuple = SearchSysCacheCopy1(DATABASEOID, ObjectIdGetDatum(MyDatabaseId));
 	if (!HeapTupleIsValid(tuple))
@@ -897,12 +897,12 @@ EventTriggerOnLogin(void)
 		!OidIsValid(MyDatabaseId) || !MyDatabaseHasLoginEventTriggers)
 		return;
 
 	StartTransactionCommand();
 	runlist = EventTriggerCommonSetup(NULL,
-										EVT_Login, "login",
-										&trigdata, false);
+									  EVT_Login, "login",
+									  &trigdata, false);
 
 	if (runlist != NIL)
 	{
 		/*
 		 * Event trigger execution may require an active snapshot.
@@ -915,25 +915,26 @@ EventTriggerOnLogin(void)
 		/* Cleanup. */
 		list_free(runlist);
 
 		PopActiveSnapshot();
 	}
+
 	/*
-	 * There is no active login event trigger, but our pg_database.dathasloginevt was set.
-	 * Try to unset this flag.  We use the lock to prevent concurrent
-	 * SetDatatabaseHasLoginEventTriggers(), but we don't want to hang the
-	 * connection waiting on the lock.  Thus, we are just trying to acquire
-	 * the lock conditionally.
+	 * There is no active login event trigger, but our
+	 * pg_database.dathasloginevt was set. Try to unset this flag.  We use the
+	 * lock to prevent concurrent SetDatatabaseHasLoginEventTriggers(), but we
+	 * don't want to hang the connection waiting on the lock.  Thus, we are
+	 * just trying to acquire the lock conditionally.
 	 */
 	else if (ConditionalLockSharedObject(DatabaseRelationId, MyDatabaseId,
 										 0, AccessExclusiveLock))
 	{
 		/*
 		 * The lock is held.  Now we need to recheck that login event triggers
 		 * list is still empty.  Once the list is empty, we know that even if
-		 * there is a backend, which concurrently inserts/enables login trigger,
-		 * it will update pg_database.dathasloginevt *afterwards*.
+		 * there is a backend, which concurrently inserts/enables login
+		 * trigger, it will update pg_database.dathasloginevt *afterwards*.
 		 */
 		runlist = EventTriggerCommonSetup(NULL,
 										  EVT_Login, "login",
 										  &trigdata, true);
 
